extensions/toggl-track/src/api/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./me";
+export * from "./workspaces";
+export * from "./projects";
+export * from "./clients";
+export * from "./tags";
+export * from "./tasks";
+export * from "./timeEntries";
```

Comment:
I would change this to ensure better intellisense. import * as statements are harder to autocomplete, as the editor will default to finding named exports. Instead of this, we could use this file to export an object containing all methods like:

```ts
const api = {
  me,
  projects
  //... etc
}

export default api;

/*
Usage:

import api from './api';

api.projects.getWorkspaceProjects();
*/
```

Or we could `...projects` etc in the object, essentially giving us the same exports, but without the need for `import * as`

The issue with not having export * as, is that we're missing types. But anyways. I'll leave this one up to you. Just a personal preference

Reply:
What editor do you use?

Reply:
@bkeys818 vscode, and I'm used to imports automatically appearing as I type. With `import * as` that doesn't happen, and you have to manually type the import statement. From there I have autocomplete

Reply:
Would this work?

```ts
export { getMe, Me } from './me'
```

Reply:
Yup ! I think that's the best solution so far